src/codegen.cpp

```diff
@@ -1242,36 +1242,34 @@ void jl_extern_c(jl_function_t *f, jl_value_t *rt, jl_value_t *argt, char *name)
 {
     assert(jl_is_tuple_type(argt));
     Function *llvmf = jl_cfunction_object(f, rt, (jl_tupletype_t*)argt);
-    if (llvmf) {
-        // force eager emission of the function (llvm 3.3 gets confused otherwise and tries to do recursive compilation)
-        uint64_t Addr = getAddressForFunction(llvmf);
+    // force eager emission of the function (llvm 3.3 gets confused otherwise and tries to do recursive compilation)
+    uint64_t Addr = getAddressForFunction(llvmf);
 
 #if defined(USE_ORCJIT) || defined(USE_MCJIT)
-        if (imaging_mode)
-             // in the old JIT, the shadow_module aliases the engine_module,
-             // otherwise, just point the alias to the declaration
+    if (imaging_mode)
+         // in the old JIT, the shadow_module aliases the engine_module,
+         // otherwise, just point the alias to the declaration
 #endif
-            llvmf = cast<Function>(shadow_output->getNamedValue(llvmf->getName()));
+        llvmf = cast<Function>(shadow_output->getNamedValue(llvmf->getName()));
 
-        // make the alias to the shadow_module
-        GlobalAlias *GA =
+    // make the alias to the shadow_module
+    GlobalAlias *GA =
 #if JL_LLVM_VERSION >= 30800
-            GlobalAlias::create(llvmf->getType()->getElementType(), llvmf->getType()->getAddressSpace(),
-                                GlobalValue::ExternalLinkage, name, llvmf, shadow_output);
+        GlobalAlias::create(llvmf->getType()->getElementType(), llvmf->getType()->getAddressSpace(),
+                            GlobalValue::ExternalLinkage, name, llvmf, shadow_output);
 #elif JL_LLVM_VERSION >= 30700
-            GlobalAlias::create(cast<PointerType>(llvmf->getType()),
-                                GlobalValue::ExternalLinkage, name, llvmf, shadow_output);
+        GlobalAlias::create(cast<PointerType>(llvmf->getType()),
+                            GlobalValue::ExternalLinkage, name, llvmf, shadow_output);
 #else
-            new GlobalAlias(llvmf->getType(), GlobalValue::ExternalLinkage, name, llvmf, shadow_output);
+        new GlobalAlias(llvmf->getType(), GlobalValue::ExternalLinkage, name, llvmf, shadow_output);
 #endif
 
 #if defined(USE_ORCJIT) || defined(USE_MCJIT)
-        // make the alias name is valid for the current session
-        jl_ExecutionEngine->addGlobalMapping(GA, (void*)(uintptr_t)Addr);
+    // make the alias name is valid for the current session
```

Comment:
you're just changing the indentation here, but is this supposed to say "make sure the alias name is valid" or "make the alias name valid" ?